android-client/app/src/main/java/com/example/habits/data/services/UserService.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.habits.data.services
+
+import com.example.habits.data.entities.network.UserDTO
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface UserService {
+    @POST("users/sign_in")
```

Comment:
have you tried it on with [fake backend server](https://github.com/espoo-dev/group-habits/tree/main/fake-json-server)?

Reply:
I mocked the user object in tests